### DISCUSSION TITLE
Consider renaming transformation output "CDM" files

### DISCUSSION THREAD

**Discussion by jsstevenson** [cancervariants/metakb#364] [Ideas]
I think there's potential for overlap/confusion with the [OMOP Common Data Model (CDM)](https://www.ohdsi.org/data-standardization/) -- and, in fact, I wonder if this was partial inspiration for the name in the first place. I'm not sure if it's a particularly helpful name, anyway (in what way is the model "Common"?)

Instead, I wonder if there's some reference we can derive from the GA4GH GKS workstream and its associated projects. We're essentially producing output that's fully consistent with the products established there, so maybe we can use a different name to indicate this.

**Comment by korikuzma** [comment]
@ahwagner @larrybabb Can you two provide input on this?

**Reply by ahwagner** [reply]
[Common Data Models](https://medinform.jmir.org/2023/1/e45116) are [a common thing](https://www.healthit.gov/sites/default/files/page/2020-07/CDMH-Project-Summary.pdf) in clinical informatics. The colloquial "common data model" was just a way for us to describe the model we were sharing across the ClinGen and VICC projects to support the GK-Pilot; at various times it has included draft models for VRS, VRSATILE, Cat-VRS, and VA-Spec.

That said, the Variant Annotation Data Elements Representation (VADER) would be a great and instantly recognizable acronym for a data model that would never be confused with the OMOP CDM.

**Reply by korikuzma** [reply]
VADER works for me. @jsstevenson what do you think?
